src/main/java/edu/ucsb/cs156/example/controllers/UCSBCurriculumController.java

```diff
@@ -0,0 +1,27 @@
+package edu.ucsb.cs156.example.controllers;
+
+import com.fasterxml.jackson.core.JsonProcessingException;
+import org.springframework.beans.factory.annotation.Autowired;
+
+import org.springframework.http.ResponseEntity;
+import org.springframework.web.bind.annotation.GetMapping;
+import org.springframework.web.bind.annotation.RequestMapping;
+import org.springframework.web.bind.annotation.RequestParam;
+import org.springframework.web.bind.annotation.RestController;
+import edu.ucsb.cs156.example.services.UCSBCurriculumService;
+
+@RestController
+@RequestMapping("/api/public")

```

Comment:
I don't think this is a good base URL for this controller. This is a UCSB Curriculum controller, not a public controller. Maybe a base URL like `/api/curriculum` would be better?

Reply:
Agreed.